Library/Homebrew/extend/os/linux/extend/ENV/super.rb

```diff
@@ -37,6 +37,17 @@ def homebrew_extra_paths
     paths
   end
 
+  def homebrew_extra_isystem_paths
+    paths = []
+    # Add paths for GCC headers when building against glibc@2.13 because we have to use -nostdinc.
+    if deps.any? { |d| d.name == "glibc@2.13" }
```

Comment:
@danielnachun I'd really like if we can extract all these references into a global constant or, if that doesn't allow superenv handling, a file somewhere.

Reply:
So basically use a variable instead of hard coding `glibc@2.13` so we only have to change it once?  That does seem better for maintainability, I just have to see where I can define it that will be seen by both the superenv and the compiler shim.

Reply:
@danielnachun I think the answer may be "nowhere" but at least if it's in two places with a comment in both that says to change them both at once that'd be better!

Reply:
Our compiler shim doesn't load the `Homebrew` module, but we can definitely put this into a global constant which is passed to the compiler shim via an environment variable.

Reply:
ooo environment variable, good idea 👍🏻

Reply:
Or the existing cccfg system

Reply:
I think we could `append_to_cccfg` a character for whenever `glibc@#{MIN_GLIBC_VERSION}` is a dependency where `MIN_GLIBC_VERSION` is an environment variable we set once globally.  The individual conditionals on `glibc@2.13` being a dependency could then be replaced with conditional on `HOMEBREW_CCCFG` containing the relevant character, and the actual flags we set would also use `#{MIN_GLIBC_VERSION}` instead of the hardcoded `2.13` we're currently using.  That should mean that whenever the version needs to be changed, we can just update `MIN_GLIBC_VERSION` and everything else will be updated automatically because it's not hardcoded.

Now that we know our strategy for handling `glibc@2.13` works I can just test the refactoring by trying to build `binutils` as it will fail quickly if I mess something up.  Thanks for the great suggestions for this!